### ISSUE TITLE
DOC: <Issue related to /docs/integrations/callbacks/datadog_tracer/>

### ISSUE THREAD

**Issue by sabrenner** [langchain-ai/langchainjs#7201]
### Checklist

- [X] I added a very descriptive title to this issue.
- [X] I included a link to the documentation page I am referring to (if applicable).

### Issue with current documentation:

Datadog LLM Observability has recently released a [Node.js SDK](https://docs.datadoghq.com/llm_observability/setup/sdk/nodejs/), and along with that are some upcoming changes to automatically instrument LangChain. It would make this LangChain community Datadog LLM Observability callback documentation obsolete: https://js.langchain.com/docs/integrations/callbacks/datadog_tracer.

### Idea or request for content:

It'd be good for this page to point to the official Datadog documentation instead (similar to our Python SDK counterpart: https://python.langchain.com/docs/integrations/providers/datadog/) once the auto-instrumentation is released.

**Comment by barieom** [comment]
Hi @bracesproul - any idea when this could request can be handled?

**Comment by bracesproul** [comment]
@barieom we would love a community contribution for this!

**Comment by ianwoodfill** [comment]
The issue with deprecating the community callback is that the official package does not currently support ESM, which is likely a large percentage of the user base.

**Comment by sabrenner** [comment]
@ianwoodfill this is a very fair point. we're looking to fix that compatibility soon. once we do, I can update here and revisit this to deprecate the community callback with new documentation.

**Comment by ianwoodfill** [comment]
Much appreciated, @sabrenner. Looking forward to being able to use dd-trace directly.

**Comment by sabrenner** [comment]
Hey folks, I wanna follow up here - we released ESM compatibility in [dd-trace@5.38.0](https://github.com/DataDog/dd-trace-js/releases/tag/v5.38.0), so we should be unblocked here for deprecating the community callback. I'll get back on writing a new doc for using Datadog LLM Observability with LangChain 😄  

@bracesproul I want to bump my question [here](https://github.com/langchain-ai/langchainjs/issues/7201#issuecomment-2610561340) - where would you suggest placing this documentation (I sorta doubt it should live under `Callbacks`). Thanks in advance!

**Comment by christian-bromann** [comment]
> I want to bump my question [here](https://github.com/langchain-ai/langchainjs/issues/7201#issuecomment-2610561340) - where would you suggest placing this documentation (I sorta doubt it should live under `Callbacks`). Thanks in advance!

@sabrenner thanks for getting back to us and sorry about the delay. We have moved our docs to a [new place](https://github.com/langchain-ai/docs/issues) and would appreciate any form of contribution there. Thank you!

**Comment by sabrenner** [comment]
@christian-bromann thank you! i'll open a new PR there shortly 😄
